### ISSUE TITLE
Multiple Inaccuracy Issues with FISH_QUEST type (and MCMMO Hook)

### ISSUE THREAD

**Issue by NolanMC** [JCulley/ProQuests-Documentation#79]
Spigot Link: https://www.spigotmc.org/members/cultivatemc.112281/
Spigot Version: This server is running Paper version git-Paper-1270 (MC: 1.12.2) (Implementing API version 1.12.2-R0.1-SNAPSHOT)
Fishing Quest: https://paste.md-5.net/mevolovuto.sm

I am trying to create a Fishing quest for my server. I am using the following quest I have pasted up above. I want players to be able to fish for the plain, raw fish. The material is correct (came from Spigot's documentation). I do not want other materials to be counted into this. 

A few issues that have been happening: 
1. MCMMO Fishing Treasures are being added to the progress of the quest. I am using default fishing treasures configuration within MCMMO, and there is no version mismatch on startup with the hook and it enables properly. 

2. RAW_FISH;0s are not always being counted. It appears that the plugin only counts them sometimes, and is not fully accurate. There do not seem to be any specific conditions that would cause this to happen, and is very random in nature. 

I am reporting this because I am requesting that the Fishing Quest have improved accuracy and that it does not count fishing drops from any 3rd party plugins. I am willing to work with you to see what conditions are creating the improper counting. 

- I have not had the chance to try other Fish types
- I have not had the chance to try the Fishing Quest without MCMMO installed (though this would be useless, because I would not uninstall MCMMO on a live server).

**Comment by JCulley** [comment]
1. Next time provide your entire server log as stated in the guidelines

2. I dont understand how MCMMO is related to this 'issue' as it has nothing to do with the fishing quest type. If MCMMO takes the lowest event priority theres nothing I can do to change that.

3. The plugin compares the item material, and data value to the approved list you have provided. There is no way in which it would only track sometimes. If this is the case that means that the data value/material pairing doesn't match the item caught while fishing.

So in conclusion it most likely is due to the item having a different data value than the one provided in the allowed materials column. As per the MCMMO treasures being added theres nothing I can do, the only thing I can suggest is being very specific with your allowed/required materials.
